### ISSUE TITLE
Support hjkl and Ctrl+U/Ctrl+D in Preview mode

### ISSUE THREAD

**Issue by trietphm** [inkdropapp/inkdrop-vim#13]
It's great to have vim key-binding in Preview mode (HTML preview) as well. Actually my main use case is to navigate around the docs so having `hjkl` and `Ctrl+U`/`Ctrl+D` is good enough.
Thank~

**Comment by craftzdog** [comment]
Yeah sounds nice!

**Comment by craftzdog** [comment]
landed in v2.0.0🎉

**Comment by basyura** [comment]
I think this is correct. 

```diff
diff --git a/src/vim.js b/src/vim.js
index e20e0fb..ceb881d 100644
--- a/src/vim.js
+++ b/src/vim.js
@@ -921,10 +921,10 @@ class Plugin {
         'vim:scroll-half-screen-down': ({ target }) => {
           target.scrollTop += target.clientHeight / 2
         },
-        'vim:scroll-half-screen-up': ({ target }) => {
+        'vim:scroll-full-screen-up': ({ target }) => {
           target.scrollTop -= target.clientHeight
         },
-        'vim:scroll-half-screen-down': ({ target }) => {
+        'vim:scroll-full-screen-down': ({ target }) => {
           target.scrollTop += target.clientHeight
         },
         'vim:move-to-line': ({ target }) => {
```

**Comment by craftzdog** [comment]
@basyura Oops!! Will fix it. Thanks!

**Comment by craftzdog** [comment]
landed in v2.0.1🎉
